src/shared-components/container/style.js

```diff
@@ -2,6 +2,7 @@ import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 
 import { SPACER, BOX_SHADOWS, COLORS, MEDIA_QUERIES } from '../../constants';
+import PropTypes from 'prop-types';
```

Comment:
This is pulling up an `eslint` error for me--did the commit hooks not catch it? We should group it up with the other absolute imports.

Reply:
I'm not totally sure what happened there...